CONTRIBUTING.md

```diff
@@ -40,3 +49,105 @@ labels
 
 We also have a [slack channel](https://openssf.slack.com/archives/C03U677QD46)
 that you can reach us at and ask questions!
+
+## Contributor ladder
+
+We are excited that you want to contribute to GUAC! This contributor ladder
+outlines different contributor roles within the project, along with
+responsibilities and privileges that come with them.
+
+Since GUAC is a complex project, there are 5 topic areas of interest:
+
+- Ingestion: ingest software security metadata. Needs to write parsers for
+  documents, maintain ingestion logic, write new collectors, etc.
+- GraphQL: define the GraphQL interface used between ingestion pipeline and
+  backend, and between backend and GUAC-based applications and front-ends.
+- Backends: define efficient code to interface with database backends (Neo4j, etc.).
+- Front-end: GUAC visualisation libraries, Javascript, CSS
+- Clients: GUAC CLI, CI Checks, Policy Engines, IDE Plugins, etc.
+
+The contribution ladder is summarized in the table below:
+
+|         Role          | Responsibilities | Requirements | Privileges | GitHub access level |
+|-----------------------|------------------|--------------|------------|---------------------|
+| Community Participant | Follow [Google's Open Source Community Guidelines](https://opensource.google/conduct/) | N/A | N/A | Read (default) |
+| Reviewer              | Review contributions from other members | Prove technical expertise in at least one GUAC topic area | Can approve PRs | Write |
+| Owner                 | Set direction and priorities for topic area. Approval of contributions | Deep technical expertise in topic area. Proven record of reviews and authorship. | Slack channel invitation | Write |
```

Comment:
we don't own slack so cant put much in slack channel invitation here, maybe google groups?

Reply:
Made it more general